ops/ccs-ops-misc/load_test/common/stats.py

```diff
@@ -0,0 +1,274 @@
+"""
+Much of this file is adapted from equivalent Locust code, particularly locust.stats.StatsCSV
+"""
+from locust.stats import StatsEntry, sort_stats, PERCENTILES_TO_REPORT
+from locust.env import Environment
+import time
+import os
+import json
+from enum import Enum
+from typing import Dict, List, Optional
+import re
+from dataclasses import dataclass
+from abc import ABC, abstractmethod
+
+# botocore/boto3 is incompatible with gevent out-of-box causing issues with SSL.
+# We need to monkey patch gevent _before_ importing boto3 to ensure this doesn't happen.
+# See https://stackoverflow.com/questions/40878996/does-boto3-support-greenlets
+from gevent import monkey
+monkey.patch_all()
+import boto3
+
+PERCENTILES_TO_REPORT = PERCENTILES_TO_REPORT
```

Comment:
It's not super necessary and might be a little confusing, but I'm basically re-exporting Locust's `PERCENTILES_TO_REPORT` in `stats.py` so that consumers of `stats.py` (our code) don't need to import from another file if they want to use the default percentiles. Probably more confusing than it's worth, so I can definitely remove it if we feel it's a bit of a code smell.

Reply:
Makes sense. I should have realized that. I'd say leave it alone, maybe add a quick comment?